README.md

```diff
@@ -38,43 +41,3 @@ deployment process.
 ## Architecture
 
 The architecture with the components involved is documented [here](docs/architecture.md)
-
-## Deployment and examples
-
-### Deploy Bare Metal Operator CRDs and CRs
-
-for testing purposes only, when Bare Metal Operator is not deployed
-
-```sh
-    make deploy-bmo-cr
-```
-
-### Deploy CAPM3
-
-Deploys CAPM3 CRDs and deploys CAPI, CABPK, CACPK and CAPM3 controllers
-
-```sh
-    make deploy
-```
-
-### Run locally
-
-Runs CAPM3 controller locally
-
-```sh
-    kubectl scale -n capm3-system deployment.v1.apps/capm3-controller-manager \
-      --replicas 0
-    make run
-```
-
-### Deploy an example cluster
-
-```sh
-    make deploy-examples
-```
-
-### Delete the example cluster
-
-```sh
-    make delete-examples
-```
```

Comment:
Is it necessary to delete these? If yes, then what will be usage of those make targets, they still remain in the Makefile

Reply:
it was just moved to dev-setup.md

Reply:
okay, missed it